minidump-stackwalk/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "minidump-stackwalk"
+version = "0.3.0"
+authors = ["Ted Mielczarek <ted@mielczarek.org>"]
```

Comment:
Do feel free to add yourself in the authors list of any of these crates if you'd like!